style.css

```diff
@@ -143,10 +149,285 @@ header {
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+  margin-right: 15px;
+  font-weight: 500;
+}
+
+.multi_post_card button {
+  align-self: flex-start;
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.7rem;
+  border: none;
+  margin-top: 12px;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.project_card {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 386px;
+  width: 327px;
+  background-image: url(./assets/project_image.svg);
+  padding-top: 80px;
+}
+
+.project_card h4 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-left: 24px;
+}
+
+.project_card p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 24px;
+}
+
+.techs_used_project {
+  display: flex;
+}
+
+.techs_used_project p {
+  margin-top: 4px;
+  margin-left: 24px;
+  padding: 10px 12px;
+  background-color: rgba(255, 255, 255, 0.24);
+  font-weight: 500;
+}
+
+.project_card button {
+  padding: 12px;
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.7rem;
+  border: none;
+  margin-top: 26px;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.about_me_section {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  padding: 24px;
+  align-items: flex-start;
+  background-image: url(./assets/background_img_about_me.svg);
+  background-repeat: no-repeat;
+  background-position: right 92%;
+}
+
+.title_line_two {
+  width: 327px;
+  height: 1px;
+  background-color: #fff;
+  margin-top: 200px;
+}
+
+.about_me_section h4 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  color: #ff6b00;
+  margin-top: 100px;
+}
+
+.about_me_section p {
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-top: 12px;
+}
+
+.about_me_section button {
+  align-self: flex-start;
+  padding: 12px;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.7rem;
+  border: none;
+  margin-top: 24px;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+.project_card button:hover {
+  background: #ff6a00bd;
+  border: solid #ff6b00;
+  border-width: 2px;
+}
+
+.multi_post_card button:hover {
+  background: #b14a00d0;
+  border: solid #ff6b00;
+  border-width: 2px;
+}
+
+.about_me_section button:hover {
+  background: #b14a00d0;
+  border: solid #ff6b00;
+  border-width: 2px;
+}
```

Comment:
- [ ] Kindly implement `Pressed` and `Active` interactions for these elements, as they are part of the requirements for the buttons in this portfolio. 👍